src/draw/lv_draw_dither.h

```diff
@@ -0,0 +1,71 @@
+/**
+ * @file lv_draw_dither.h
+ *
+ */
+
+#ifndef LV_DRAW_DITHER_H
+#define LV_DRAW_DITHER_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "../core/lv_obj_pos.h"
+
+
+/*********************
+ *      DEFINES
+ *********************/
+#if LV_COLOR_DEPTH < 32 && LV_DRAW_COMPLEX == 1 && LV_DITHER_GRADIENT == 1
+#define DITHER_GRADIENT 1
```

Comment:
It should be prefixed with an underscore to avoid possible conflicts with user code.

Reply:
Ok